src/client/app/translations/data.js

```diff
@@ -582,7 +580,11 @@ const localeData = {
 		"DisplayableType.none": "(Need French) none",
 		"DisplayableType.all": "(Need French) all",
 		"DisplayableType.admin": "(Need French) admin",
-		"duplication.invalid": "(Need French) The reading duplication must be an integer between 1 and 9 but you gave ",
+		"error.bounds": "(Need French) Must be between {min} and {max}.",
+		"error.displayable": "(Need French) Displayable will be set to false.",
+		"error.greater": "(Need French) Must be greater than {min}.",
+		"error.negative": "(Need French) Cannot be negative.",
```

Comment:
You did not do this but while checking I realized error.gps is missing after this. Would you be willing to add it? Also missing from Spanish.